src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/MethodWriter.java

```diff
@@ -105,13 +105,13 @@ protected void buildMethodDoc(Content detailsList) {
             for (Element method : methods) {
                 currentMethod = (ExecutableElement)method;
                 Content methodContent = getMethodHeader(currentMethod);
-
-                buildSignature(methodContent);
-                buildDeprecationInfo(methodContent);
-                buildPreviewInfo(methodContent);
-                buildMethodComments(methodContent);
-                buildTagInfo(methodContent);
-
+                Content div = HtmlTree.DIV(HtmlStyle.horizontalScroll);
+                buildSignature(div);
+                buildDeprecationInfo(div);
+                buildPreviewInfo(div);
+                buildMethodComments(div);
+                buildTagInfo(div);
+                methodContent.add(div);
```

Comment:
Purely as a matter of style, I like the style of building "bottom up", so first build all those items into a `ContentBuilder`, then make a `div` with `HtmlTree.DIV(HtmlStyle.horizontalScroll, contentBuilder)`  then add the `div` into the `methodContent`, as on line 114.